docs/meshstack.aws.metering.md

```diff
@@ -70,15 +70,18 @@ In order for meshStack to generate Usage Reports, following policies are require
 
 ## Reserved Instances & Savings Plans
 
-meshStack collects the information about reserved instances and savings plans via the cost explorer API.
+This section applies only if your meshCustomers buy Reserved Instances and Savings Plans which give priority to their
+AWS tenants. If this is the case you can enable `reservedInstanceFairChargeback` and `savingsPlanFairChargeback` feature
+flags to achieve the following.
```

Comment:
I think the text would be improved by linking the RI guide already here and mentioning that it explains why an operator may want to configure this.

The way you link it at the end of the section sounds more like it would be a technical reference about cost explorer api (which it's not :-) )

Reply:
Done.